src/elastio-snap.h

```diff
@@ -10,6 +10,7 @@
 
 #ifndef __KERNEL__
 #include <stdint.h>
+#include <stdbool.h>
```

Comment:
In kernel  **int** was used usually. Is it compatible?

Reply:
Yes, it is.

Reply:
Also it's defined for the linking from userspace. See line 11 `#ifndef __KERNEL__`.